docs/Regenerating Server and Client.md

```diff
@@ -1,115 +1,115 @@
 # Regenerating Servers and Clients
 
-Below are instructions for regenerating the grpc-labview code for MeasurementLink servers and clients.
+Below are instructions for regenerating the grpc-labview code for measurement plug-in servers and clients.
 
 ## Setup
 
 - Install LabVIEW 2020 SP1
 - Install all `grpc-labview` VI packages from the desired `grpc-labview` [release](https://github.com/ni/grpc-labview/releases)
-- Clone the `measurementlink-labview` github repo
+- Clone the `measurement-plugin-labview` github repo
 
-## MeasurementLink Discovery Client
+## Discovery Client
 
 The following are instructions for regenerating the Discovery Client.
 
-- Open `\Source\Runtime\Discovery Client.lvproj`
-- Delete the entire `MeasurementLink Discovery Client` virtual folder and save the project
-- Delete the entire `MeasurementLink Discovery Client` on disc
+- Open `\Source\gRPC\Generated APIs\ni\measurementlink\discovery\v1\ni.measurementlink.discovery.v1.api.lvproj`
+- Delete the entire `ni.measurementlink.discovery.v1` virtual folder and save the project
+- Delete the entire `ni.measurementlink.discovery.v1` on disc
 - Open the gRPC Template Creation Utility. `Tools` » `gRPC` » `Open gRPC Server-Client [2] - Code Generator...`
 - Fill out the utility as follows:
     - **Proto File Path:** `\Source\Protos\ni\measurementlink\discovery\v1\discovery_service.proto`
     - **Import Paths:** `\Source\Protos`
-    - **Prototype Library:** `\Source\Runtime\Discovery Client Prototype Library.lvlib`
-    - **Target Project:** `\Source\Runtime\Discovery Client.lvproj`
+    - **Prototype Library:** `\Source\gRPC\Generated APIs\ni\measurementlink\discovery\v1\ni.measurementlink.discovery.v1.prototype.lvlib`
+    - **Target Project:** `\Source\gRPC\Generated APIs\ni\measurementlink\discovery\v1\ni.measurementlink.discovery.v1.api.lvproj`
     - **Target Name:** `My Computer`
-    - **Generated Library Name:** `MeasurementLink Discovery`
-    - **Generated Library Suffix:** `Client` (make sure to put a space at the beginning)
+    - **Generated Library Name:** `ni.measurementlink.discovery.v1`
+    - **Generated Library Suffix:** `.api`
```

Comment:
Totally guessing on what to put in these "Generated Library" lines, based on what was there before and the before/after file names.